### ISSUE TITLE
does not seem to work on the newly updated clickerheros anymore

### ISSUE THREAD

**Issue by fawaf** [Legocro/CH-save-editor#4]

**Comment by Snowpup** [comment]
Yeah I really wish this could be updated, only just started the game and all the tools out there are now defunct D:

Edit: For anyone else out there that manages to check the Git issues after finding this one not working, this editor does work and has way more options but it's way less friendly to actually use.

https://clickerheroes.hd-gaming.eu/
